script.js

```diff
@@ -44,5 +47,35 @@ function addTeamMember(event) { //eventlistener
     renderStats();
 }
 
+//below is light/dark toggle for the page, new consts will be at the top.
+
+function lastKnownMode () {
+    document.body.className = mode;
+    if (mode === 'light') {
+      toggleBtn.textContent = '☀️';
+    } else {
+      toggleBtn.textContent = '🌙';
+    }
+  }
+  
+  function darkModeToggle() {
```

Comment:
I think we probably want to start this darkmode toggle function by checking the local storage value first - but I guess that depends on if we want darkmode to persist to the next session (I think we do)